ehr/resources/pipeline/kinship/populateInbreeding.r

```diff
@@ -41,6 +45,9 @@ ib = calcInbreeding(df);
 
 newRecords <- data.frame(Id=as.character(df$id), coefficient=ib, stringsAsFactors=FALSE);
 
+#only calculate inbreeding for Ids at the center
+newRecords <- dplyr::filter(newRecords, Id %in% originalIds$id)
```

Comment:
this doesnt really matter that much but you could do this without dplyr.

Reply:
Yeah I figured you could, dplyr is just a proven way to do it efficiently so I went with that solution. Also using dplyr in populateKinship.